src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -0,0 +1,295 @@
+                                                                              ToolTip="{x:Static p:Resources.InstalledPackageViewContextMenuPublishTooltip}"
+                                                                              Header="{x:Static p:Resources.InstalledPackageViewContextMenuPublish}">
+                                                            </MenuItem>
+                                                            <MenuItem Name="MakeNewVersionButton"
+                                                                              Command="{Binding Path=PublishNewPackageVersionCommand}"
+                                                                              ToolTip="{x:Static p:Resources.InstalledPackageViewContextMenuPublishVersionTooltip}"
+                                                                              Header="{x:Static p:Resources.InstalledPackageViewContextMenuPublishVersion}">
+                                                            </MenuItem>
+                                                            <MenuItem Name="DeprecateButton"
+                                                                              Command="{Binding Path=DeprecateCommand}"
+                                                                              ToolTip="{x:Static p:Resources.InstalledPackageViewContextMenuDeprecateTooltip}"
+                                                                              Header="{x:Static p:Resources.InstalledPackageViewContextMenuDeprecate}">
+                                                            </MenuItem>
+                                                            <MenuItem Name="UndeprecateButton"
+                                                                              Command="{Binding Path=UndeprecateCommand}"
+                                                                              ToolTip="{x:Static p:Resources.InstalledPackageViewContextMenuRemoveDeprecationTooltip}"
+                                                                              Header="{x:Static p:Resources.InstalledPackageViewContextMenuRemoveDeprecation}">
+                                                            </MenuItem>
+                                                        </ContextMenu>
+
+                                                    </Button.ContextMenu>
+                                                </Button>
+                                            </StackPanel>
+                                        </DockPanel>
+                                    </Grid>
+                                </Expander.Header>
+                                <Grid>
+                                    <Border BorderThickness="0,0,0,0"
+                                                BorderBrush="#111"
+                                                Background="#3c3c3c"
```

Comment:
this is a very similar color to:
https://github.com/DynamoDS/Dynamo/blob/bdd63b69c365d5ce5b07ff4ef20649983d49e074/src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml#L2272

maybe we can just use that?

Reply:
The caret expander has different colors. I pulled out the colors from the style in `DynamoModern.xaml` so we can reuse them here.